### PR TITLE
Reuse onpremis instance

### PR DESCRIPTION
**Pull request by mahendranbhat**
Internal review

### CODE REVIEW

File: managed/src/main/java/com/yugabyte/yw/controllers/UniverseController.java [hunk 1]
```diff
@@ -1004,7 +1005,18 @@ public Result destroy(UUID customerUUID, UUID universeUUID) {
     if (primaryCluster.userIntent.providerType.equals(CloudType.kubernetes)) {
       taskType = TaskType.DestroyKubernetesUniverse;
     }
-
+	if (primaryCluster.userIntent.providerType.equals(CloudType.onprem)) {
```
Comment (jitendra-12113) [inline managed/src/main/java/com/yugabyte/yw/controllers/UniverseController.java:1008]: need one more space

File: managed/src/main/java/com/yugabyte/yw/controllers/UniverseController.java [hunk 2]
```diff
@@ -1004,7 +1005,18 @@ public Result destroy(UUID customerUUID, UUID universeUUID) {
...
+		Set<NodeDetails> universeDetailList = universeDetails.nodeDetailsSet;
+		for (NodeDetails eachNode : universeDetailList) {
+			NodeInstance nodeInstanceObject = NodeInstance.find.query().where().eq("nodeName", eachNode.nodeName).findOne();
+			if (nodeInstanceObject != null) {
```
Comment (jitendra-12113) [inline managed/src/main/java/com/yugabyte/yw/controllers/UniverseController.java:1012]: shouldn't we handle isEmpty()?

**Comment by jitendra-12113** [comment]
Please add Unit tests.

**Review by jitendra-12113 — COMMENTED** [review]
Please fix indentation format. Also please add UTs,

**Comment by mahendranbhat** [comment]
The above code resides in subtasks. Closing the pull request and adding fix in new PR as this PR is from my repository.

**Pull request closed without merging**